libraries/mbed/targets/hal/TARGET_NORDIC/TARGET_MCU_NRF51822/TARGET_DELTA_DFCM_NNN40/mbed_overrides.c

```diff
@@ -115,9 +118,11 @@ void mbed_sdk_init()
     {// Do nothing.
     }
 	
-	flash_init();
-	
-	//nrf_delay_ms(10);
-	flash_powerDown();
+/* No init flash in this version, 2015/10/27 */
```

Comment:
You can compare the latest master when you start creating a new PR, can check if these files are shown there. IF no, close this one and open a new one.